tasty-plutus/src/Test/Tasty/Plutus/Internal/Context.hs

```diff
@@ -124,14 +149,19 @@ data Input
 -}
 data Output
   = -- | @since 3.0
-    Output ExternalType Value
+    Output ExternalType ValueType
```

Comment:
Since this constructor is exported, this is an API change. Therefore, the `since` needs to match the version where it changed.